R/sensitivity-analysis-results.R

```diff
@@ -53,6 +57,16 @@ SensitivityAnalysisResults <- R6::R6Class("SensitivityAnalysisResults",
       toObjectType(pkParameterSentitivities, PKParameterSensitivity)
     },
     #' @description
+    #' Returns the sensisivity value for a given pkParameter, output and parameter. If the sensitivity result does not exist, returns `NaN`
+    #' @param pkParameterName Name of pkParameter for which sensitivity should be retrieved
+    #' @param outputPath Path of the output for which the sensitivity should be retrieved
+    #' @param parameterName Name of the sensitivity parameter for which the sensitivity should be retrieved
+    pkParameterSensitivityValueFor = function(pkParameterName, outputPath, parameterName) {

```

Comment:
@abdullahhamadeh This is the function that returns the value for the combo {PK, path, parameterName }